Loenn/lang/en_gb.lang

```diff
@@ -0,0 +1,43 @@
+# Mod Name
+mods.EverestYamlName.name=IsaGrabBag
+
+# Arrow Bubble
+entities.isaBag/arrowBubble.placements.name.default=arrow Bubble
```

Comment:
please use consistent capitalization for these lol. so "arrow Bubble" -> "Arrow Bubble"